taskrunner/host_gateway.go

```diff
@@ -229,6 +233,9 @@ func startAgentOnRemote(settings *evergreen.Settings, hostObj *host.Host, sshOpt
 		fmt.Sprintf("--log_prefix='%s'", filepath.Join(hostObj.Distro.WorkDir, agentFile)),
 		fmt.Sprintf("--working_directory='%s'", hostObj.Distro.WorkDir),
 	}
+	if hostObj.Distro.SetupAsSudo {
+		agentCmdParts = append(agentCmdParts, "--setup_as_sudo")
+	}
 
 	// build the command to run on the remote machine
 	remoteCmd := strings.Join(agentCmdParts, " ")
```

Comment:
On the logging line below this, can you add message.fields to give it more context? Maybe with hostID=host ID and message="running remote script on agent"

Reply:
done